protocol/x/clob/memclob/memclob.go

```diff
@@ -1696,9 +1696,10 @@ func (m *MemClobPriceTimePriority) mustPerformTakerOrderMatching(
 			FillAmount: matchedAmount,
 		}
 
-		affiliatesWhitelistMap := make(map[string]uint32)
+		// affiliate revshare doesnt affect subaccount balance, we can pass an empty map
+		// as we dont need to process affiliate revshare during checkTx and only during deliverTx
 		success, takerUpdateResult, makerUpdateResult, _, err := m.clobKeeper.ProcessSingleMatch(
-			ctx, &matchWithOrders, affiliatesWhitelistMap)
+			ctx, &matchWithOrders, make(map[string]uint32))
```

Comment:
Nit: pass in literal for readability
```suggestion
			ctx, &matchWithOrders, map[string]uint32{})
```